README.md

```diff
@@ -35,7 +35,7 @@ If you want to modify *a single file* in the core library (not the C++ source),
 ```
 git checkout v3.xx.x
 git checkout -b my-branch-name
-elan override set leanprover-community-lean-3.xx.x
+elan override set leanprover-community/lean:v3.xx.x
```

Comment:
```suggestion
elan override set leanprover-community/lean:3.xx.x
```

Both the v- and v-less versions work, but elan thinks they're different toolchains and will download them twice.  Since we use the v-less version in mathlib, we should also use the v-less one here.

Reply:
Oh, I didn't know that. Changed.